features/file-operations.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Basic File operations
+  As a user
+  I want to be able to do basic file operations
+  So that I can organize my files
+
+  Background: Space Exists
+    Given user "marie" has logged in with password "radioactivity"
+    Given user "marie" has created a personal space with the alias "Maries Home"
```

Comment:
```suggestion
    And user "marie" has created a personal space with the alias "Maries Home"
```

Reply:
please also change the step implementation